### ISSUE TITLE
Conformer  decode speed too slow 

### ISSUE THREAD

**Issue by Zepan** [TensorSpeech/TensorFlowASR#58]
hi, I'm using Conformer  example, training is good, but the decode speed is very slow.
I tried another speech-transform based project: https://github.com/ZhengkunTian/OpenTransformer
use same GPU, almost same size model (~30M parm), 
TensorFlowASR's Conformer   infer 3 wav/s, OpenTransformer's speech-transform infer 20 wav/s
I look at TensorFlowASR's code, it infer one wav per batch, is it the reason why it is so slow? do you have plan to improve it?

**Comment by usimarit** [comment]
@Zepan In my opinion, testing on 1 wav at a time gives in more realistic results than the whole batch, because it don't require "padding" the whole batch to match the longest wav in the batch, that's why I don't use batch size > 1 for inference

**Comment by Zepan** [comment]
Ok, I see.  but I'm doing KD now, it need infer huge dataset...  
and also doing standard testset testing will cost lot time.
I will add the bs>1 feature and PR it.

**Comment by Zepan** [comment]
@usimarit , just test bs=2, I find the cer will increase notable (5.5% -> 9.1% in aishell)
and I check the output, there are some addition error output in the end of the sentence.
seems how long it padded, the addtion output is that long too...
It seems network don't understand how to deal with padding? how to solve it?

example:
/mnt/wd8T/dataset/kws/data_aishell/wav/test/S0904/BAC009S0904W0416.wav	
GT:         悇愓悂抄态怡怐	
nopad:   悇愓悂抄态怡怆	
withpad 悇愓悂抄态怡怆恸愗		#last two char is error addition output

**Comment by usimarit** [comment]
I think the network don't understand padding is because of specaugment, the augmentation masks part of spectrogram features, and the network tried to reconstruct the masked parts, and the padding is like the mask, so the network tries to reconstruct that part to, therefore 2 characters at the end appeared.

**Comment by Zepan** [comment]
ok, at last I use multiprocess to accelerate the infer speed...
